odoo/addons/base/migrations/12.0.1.3/end-migration.py

```diff
@@ -46,6 +46,15 @@ def fork_off_system_user(env):
         'partner_id': partner_admin.id,
         'login': login,
     })
+    # copy old passwords for not losing them on new admin user
+    set_query = "SET password = ru2.password "
+    if openupgrade.column_exists(env.cr, 'res_users', 'password_crypt'):
+        set_query += ", password_crypt = ru2.password_crypt "
```

Comment:
Thanks! password_crypt is guaranteed to exist as per 11.0 datamodel, as is also assumed down below in the `migrate` method. Can you set password_crypt to NULL in https://github.com/OCA/OpenUpgrade/pull/1801/files#diff-dc09a830b46c834278b85d839da79ba1L91?

Reply:
@StefanRijnhart not really, as although `auth_crypt` module, which adds this field, is auto-installed by default, there's a chance that you manually uninstall it afterwards. About the other request, there's no need of doing that, as `password_crypt` is no longer used in v12.

Reply:
@pedrobaeza the reason I ask to empty `password_crypt` is because later on in the same script, its value is copied to `password`: https://github.com/OCA/OpenUpgrade/blob/12.0/odoo/addons/base/migrations/12.0.1.3/end-migration.py#L100. It is also there where the existence of the column is assumed so your check is useless unless you also implement the check there (or leave it out as I asked you earlier, but I don't really mind which way).

Reply:
OK, I see. Fixed.

Reply:
Thanks! I see I was wrong about the existence of this column being assumed in fill_res_users_password_from_password_crypt because it is checked before this method is called. Sorry for the confusion.